OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -60,6 +69,12 @@ public override void Draw()
 				isHover ? BaseName + "-hover" :
 				null;
 
+			if (DrawMode == ScrollItemDrawMode.Background && state != null)
```

Comment:
Thinking it would be better to make a new widget with this behavior instead of making these changes in ScrollItemWidget